Source/Core/VideoCommon/VertexShaderGen.cpp

```diff
@@ -23,7 +23,6 @@ VertexShaderUid GetVertexShaderUid()
   memset(uid_data, 0, sizeof(*uid_data));
 
   _assert_(bpmem.genMode.numtexgens == xfmem.numTexGen.numTexGens);
-  _assert_(bpmem.genMode.numcolchans == xfmem.numChan.numColorChans);
```

Comment:
Potentially. The issue here is we're not sure how the console would behave in this situation, we might be able to get away with using zero in such cases, but it would be better to test this.